test/scapy/layers/dot15d4.uts

```diff
@@ -816,3 +844,23 @@ assert ZCLGeneralReadAttributes in pkt.layers()
 assert len(pkt[ZCLGeneralReadAttributes].attribute_identifiers) == 1
 assert pkt[ZCLGeneralReadAttributes].attribute_identifiers[0] == 0x0000
 assert raw(pkt[ZCLGeneralReadAttributes].payload) == b''
+
+= Zigbee - ZCL General command: Read Attributes Response
+
+pkt = ZCLGeneralReadAttributesResponse(b'\x00\x00\x00\x10\x00')
```

Comment:
Could you make your tests have slighly less default values? Thanks !

Reply:
Sure! I modified some of the field values and squashed the new commit.